cinema/urls.py

```diff
@@ -18,6 +17,8 @@
 router.register("movie_sessions", MovieSessionViewSet)
 router.register("orders", OrderViewSet)
 
-urlpatterns = [path("", include(router.urls))]
+urlpatterns = [
+    path("", include(router.urls)),
+]
```

Comment:
Consider adding a specific endpoint for image uploads to handle image uploads for the `Movie` model, as required by the task description.